testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -751,6 +720,21 @@ class _GromacsReader_offsets(TestCase):
     ref_volume = 362270.0
     ref_offsets = None
     _reader = None
+    prec = 3
+    
+    @pytest.fixture()
+    def traj(self, tmpdir):
+        shutil.copy(self.filename, str(tmpdir))
+        return os.path.join(str(tmpdir), os.path.basename(self.filename))
+    
+    @pytest.fixture()
+    def trajectory(self, traj):
+        return self._reader(traj)
+
+    @pytest.fixture()
```

Comment:
The `ts` fixture seems overkill; I don't think it is wrong but looks overly complicated. I would rewrite the tests to use `trajectory.ts` instead of `ts`.